code/backend/src/main/java/server/rest/controllers/HRPayGradeController.java

```diff
@@ -2,23 +2,28 @@
 
 import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 import server.database.DatabaseConnection;
 import server.model.HRPayGrade;
+import server.rest.responses.HRAddEditPayGradeResponse;
 import server.rest.responses.HRPayGradeResponse;
 import server.rest.responses.Response;
 
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.ArrayList;
+import java.util.UUID;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 @CrossOrigin(origins = {"http://localhost:1234","http://theterminal.s3-website.us-west-2.amazonaws.com"})
 @RestController
 public class HRPayGradeController extends Controller {
     private static String getQuery = "select * from HRPayGrade;";
+    private static String addQuery = "insert into HRPayGrade values(?, ?, ?);";
```

Comment:
This might not work due to the semicolon at the end of the query. I don't think you're supposed to add a semicolon when using PreparedStatement (as it adds one itself if I recall correctly). However I'm not sure if it still works with a semicolon so maybe test it out by hitting the endpoint in a browser or in PostMan.

Reply:
if you haven't already of course.

Reply:
if you have then its fine 👍